src/libraries/Common/tests/System/Net/Capability.Security.Unix.cs

```diff
@@ -7,9 +7,7 @@ public static partial class Capability
     {
         public static bool IsNtlmInstalled()
         {
-            // GSS on Linux does not work with OpenSSL 3.0. Fix was submitted to gss-ntlm but it will take a while to make to
-            // all supported distributions. The second part of the check should be removed when it does.
-            return Interop.NetSecurityNative.IsNtlmInstalled() && (!PlatformDetection.IsOpenSslSupported || PlatformDetection.OpenSslVersion.Major < 3);
+            return true;
```

Comment:
Do we need the property at all? Or keep it for future?

Reply:
I guess that depends on the answer in https://github.com/dotnet/runtime/pull/87930#discussion_r1254905782. In any case I would not remove it as part of this PR, it would be separate change. Mainly there's a concern about Windows with NTLM disabled, so there would still be use for it.